superset-frontend/src/components/Tags/Tag.stories.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { AntdThemeProvider } from 'src/components/AntdThemeProvider';
+import TagType from 'src/types/TagType';
+import { Tag } from '.';
+
+export default {
+  title: 'Tags',
+  component: Tag,
+};
+export const InteractiveTag = (args: TagType) => (
+  <AntdThemeProvider>
```

Comment:
In https://github.com/apache/superset/pull/29545 I add the theme provider globally, so that we don't need to do it in every story :)